templates/editor.tera

```diff
@@ -572,28 +574,28 @@ whiskers:
         <WidgetStyle name="Global override" styleID="0" fgColor="{{ text.hex }}" bgColor="{{ base.hex }}" fontName="Consolas" fontStyle="0" fontSize="10" />
         <WidgetStyle name="Default Style" styleID="32" fgColor="{{ text.hex }}" bgColor="{{ base.hex }}" fontName="Consolas" fontStyle="0" fontSize="10" />
         <WidgetStyle name="Indent guideline style" styleID="37" fgColor="{{ blue.hex }}" bgColor="{{ base.hex }}" fontName="" fontStyle="0" fontSize="" />
-        <WidgetStyle name="Brace highlight style" styleID="34" fgColor="{{ text.hex }}" bgColor="{{ mauve.hex }}" fontName="" fontStyle="1" fontSize="10" />
+        <WidgetStyle name="Brace highlight style" styleID="34" fgColor="{{ text.hex }}" bgColor="{{ selected.hex }}" fontName="" fontStyle="1" fontSize="10" />
         <WidgetStyle name="Bad brace colour" styleID="35" fgColor="{{ text.hex }}" bgColor="{{ red.hex }}" fontName="" fontStyle="0" fontSize="" />
-        <WidgetStyle name="Current line background colour" styleID="0" bgColor="2F3142" fgColor="{{ text.hex }}" fontSize="" fontStyle="0" />
-        <WidgetStyle name="Selected text colour" styleID="0" bgColor="44475A" fgColor="{{ text.hex }}" fontStyle="0" />
-        <WidgetStyle name="Caret colour" styleID="2069" fgColor="{{ text.hex }}" bgColor="{{ base.hex }}" fontStyle="0" />
+        <WidgetStyle name="Current line background colour" styleID="0" bgColor="{{ line.hex }}" fgColor="{{ text.hex }}" fontSize="" fontStyle="0" />
+        <WidgetStyle name="Selected text colour" styleID="0" bgColor="{{ selected.hex }}" fontStyle="0" />
+        <WidgetStyle name="Caret colour" styleID="2069" fgColor="{{ rosewater.hex }}" bgColor="{{ base.hex }}" fontStyle="0" />
         <WidgetStyle name="Edge colour" styleID="0" fgColor="{{ text.hex }}" bgColor="{{ base.hex }}" fontSize="" fontStyle="0" />
-        <WidgetStyle name="Line number margin" styleID="33" fgColor="{{ mauve.hex }}" bgColor="{{ base.hex }}" fontName="" fontStyle="0" fontSize="" />
+        <WidgetStyle name="Line number margin" styleID="33" fgColor="{{ overlay1.hex }}" bgColor="{{ base.hex }}" fontName="" fontStyle="0" fontSize="" />
```

Comment:
We have some functionality in other ports where the active line number is changed to `mauve` or a chosen accent colour. 

Do you know happen to know if Notepad++ supports that? No worries if not, just wanting to understand.

Reply:
Nope, there doesn't seem to be a highlight group for active line numbers